# Wildfire raster

# Creation time
Header header

Raster raster